### PR TITLE
Adding the sift-integration test for the sift-java workflow

### PR DESCRIPTION
**Pull request by VinayakGadad**
## Purpose:
- https://sift.atlassian.net/browse/CI-11587

## Technical overview:
- Build and package the sift-java project into a distribution zip file.
- Add SSH keys with specified fingerprints for secure access.
- Clone the sift-java-integration-app repository, and extract the contents of sift-java-3.10.0.zip directly into the app/libs/ directory.
- Change to the sift-java-integration-app directory and run the tests using Gradle.

## Testing plan:
- https://app.circleci.com/pipelines/github/SiftScience/sift-java/218/workflows/3341d8a4-105c-4934-8577-0cd5b46cbc2b

## Deployment plan:
- Merge the PR.

### CODE REVIEW

**Review by btusakul-sift — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -62,11 +67,44 @@ jobs:
...
+          fingerprints:
+            - "4c:c7:1d:f2:95:87:3c:f6:61:4e:46:5a:c8:2a:d7:c9"
+
+      - run:
+          name: Clone sift-java-integration-app and extract sift-java
+          command: |
+            export GIT_SSH_COMMAND='ssh -i ~/.ssh/id_rsa_4cc71df295873cf6614e465ac82ad7c9'
+            git clone git@github.com:SiftScience/sift-java-integration-app.git
+            cd sift-java-integration-app
+            unzip -j ../build/distributions/sift-java-3.10.0.zip -d app/libs/
```
[the remark this answers is not in the document]
Comment (VinayakGadad) [reply .circleci/config.yml]: Changed it to dynamically pick the versions from gradle properties.

**Review by btusakul-sift — CHANGES_REQUESTED** [review]

**Review by mpierotti-sift — APPROVED** [review]
LGTM

File: .circleci/config.yml [hunk 2]
```diff
@@ -62,11 +67,49 @@ jobs:
...
+
+      - add_ssh_keys:
+          fingerprints:
+            - "4c:c7:1d:f2:95:87:3c:f6:61:4e:46:5a:c8:2a:d7:c9"
+
+      - run:
+          name: Clone sift-java-integration-app and extract sift-java
+          command: |
+            export GIT_SSH_COMMAND='ssh -i ~/.ssh/id_rsa_4cc71df295873cf6614e465ac82ad7c9'
+            version=$(./gradlew properties -q | grep "version:" | awk '{print $2}')
```
[the remark this answers is not in the document]
Comment (VinayakGadad) [reply .circleci/config.yml]: When I ran the `./gradlew properties -q` it had only 1 version variable. It makes sense if there are multiple matches. 
I have updated the grep accordingly.

**Review by btusakul-sift — APPROVED** [review]
LGTM

**Pull request merged**